microservices/user/app/data/feedback.go

```diff
@@ -0,0 +1,21 @@
+package data
+
+import (
+	"time"
+
+	"go.mongodb.org/mongo-driver/bson/primitive"
+)
+
+// The Feedback structure represent a feedback object in the database
+type Feedback struct {
+	ID primitive.ObjectID `json:"-" bson:"_id,omitempty"`
+
```

Comment:
Can you run go fmt on this file ?
In general it's a good practice to run fmt on every file to keep a coding style consistency.
You can set it up on Goland to run before every commit

Reply:
I did run, it's fine